test/runtests.jl

```diff
@@ -4,6 +4,23 @@ using Pkg
 
 println("HDF5 version ", HDF5.h5_get_libversion())
 
+# Copied over from JuliaLang/Test
```

Comment:
Why copy this? As far as I can tell, `Test` has exported `GenericString` since v0.5:
```bash
$ grep -ne 'export.*GenericString' stdlib/Test/src/Test.jl
23:export GenericString, GenericSet, GenericDict, GenericArray, GenericOrder
$ git log --oneline --no-patch -L 23,23:stdlib/Test/src/Test.jl
744ead41cb fix maximum on generic orders (#30441)
66ef347631 Base.Test: add GenericArray type and use it in test/random.jl (#22790)
c6799e6437 enable rand(::Union{AbstractSet,Associative}) (#22228)
0a61406e5a GenericString: move into Base.Test and export; rm CharString.
$ contrib/commit-name.sh 0a61406e5a
0.5.0-dev+4653
```

Reply:
Thanks, didint realize it was exported